http/src/main/resources/reference.conf

```diff
@@ -200,6 +200,11 @@ azure {
       max-attempts = 240 # 10 seconds * 240 is 40 min
       interval = 10 seconds
     }
+    create-disk-poll-config {
+      initial-delay = 5 seconds
+      max-attempts = 10 # 5 seconds * 10 + 5 is 55 sec
+      interval = 5 seconds
+    }
```

Comment:
Did you verify in testing that ~1min was enough time for a disk to create?

Reply:
Yea, I'll be relying on the automation tests to run this a few times (pending).

Reply:
Anecdotally, it took seconds in my manual testing